script/fix-animation

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import re
+import argparse
+
+
+def fix_animation(input: str, output: str) -> None:
+    with open(input) as f:
+        data = f.readlines()
+
+    output_data = []
+    for line in data:
+        if 'stylesheet' in line:
+            match = re.search(r'href="(.*?)"', line)
+            if match is None:
+                output_data.append(line)
+            else:
+                css_name = match[1].replace('\\', '/').split('/')[-1]
+                output_data.append(f'    <link rel="stylesheet" href="{css_name}">\n')
```

Comment:
Feels like it would be safer to do a find & replace on the adjusted text rather than fully rebuilding the line. That would avoid any issues with the underlying template changing (also below).